scavenger-collector/src/main/kotlin/com/navercorp/scavenger/service/GarbageCollectService.kt

```diff
@@ -101,9 +101,14 @@ class GarbageCollectService(
                     customerId,
                     baseDateTime.minusMillis(intervalService.batchSweepMarginMilliSecond)
                 )
-                val uuidsWithoutAgent = jvmDao.findAllUuidsByWithoutAgent(customerId)
 
-                jvmDao.deleteAllByCustomerIdAndUuids(customerId, agentStates.map { it.jvmUuid } + uuidsWithoutAgent)
+                do {
```

Comment:
Expect a lot of loops.
Instead of a nested while, how about a separate while?

Reply:
fixed in e0d780ac0e93dc2d4a9bb464888a4ae25199477d